src/Compilers/CSharp/Test/Emit/Attributes/AttributeTests.cs

```diff
@@ -3679,8 +3678,9 @@ class Class3 { }
             var compilation = CreateCompilationWithMscorlib(source);
 
             compilation.VerifyDiagnostics(
-                // (13,2): error CS0246: The type or namespace name '@X' could not be found (are you missing a using directive or an assembly reference?)
-                Diagnostic(ErrorCode.ERR_SingleTypeNameNotFound, "@X").WithArguments("@X"));
+                // (13,2): error CS0246: The type or namespace name '@X' or '@XAttribute' could not be found (are you missing a using directive or an assembly reference?)
```

Comment:
It feels like the error became misleading in this scenario, XAttribute is not going to be a suitable match.